src/scss/custom.scss

```
@@ -0,0 +1,89 @@
+/* stylelint-disable docusaurus/copyright-header */
+/**
+ * Any CSS included here will be global. The classic template
+ * bundles Infima by default. Infima is a CSS framework designed to
+ * work well for content-centric websites.
+ */
+
+ @font-face {
+  font-family: CMU-Serif;
+  src: url('/fonts/cmunbx.ttf') format('truetype');
+  style: bold;
+  weight: bold; 
+  stretch: 100;
+}
+@font-face {
+  font-family: CMU-Serif;
+  src: url('/fonts/cmunci.ttf') format('truetype');
+  style: italic;
+  weight: 400; 
+  stretch: 100;
+}
+@font-face {
+  font-family: CMU-Serif;
+  src: url('/fonts/cmunrm.ttf') format('truetype');
+  style: normal;
+  weight: 400; 
+  stretch: 100;
+}
+
+/* You can override the default Infima variables here. */
+:root {
+  // --ifm-link-color: #0044FF;
+  // --ifm-menu-color-active: #0044FF;
+  // --ifm-toc-link-color: #0044FF;
+  // --ifm-color-primary: #25c2a0;
+  --ifm-color-primary: #0044FF;
+  --ifm-color-content: var(--ifm-color-emphasis-800);
+  // --ifm-color-primary-dark: rgb(33, 175, 144);
+  // --ifm-color-primary-darker: rgb(31, 165, 136);
+  // --ifm-color-primary-darkest: rgb(26, 136, 112);
+  // --ifm-color-primary-light: rgb(70, 203, 174);
+  // --ifm-color-primary-lighter: rgb(102, 212, 189);
+  // --ifm-color-primary-lightest: rgb(146, 224, 208);
+  --ifm-code-font-size: 95%;
+}
+
+.docusaurus-highlight-code-line {
+  background-color: rgb(72, 77, 91);
+  display: block;
+  margin: 0 calc(-1 * var(--ifm-pre-padding));
+  padding: 0 var(--ifm-pre-padding);
+}
+
+.med {
+  width: 720px;
+}
+
+.caption-img {
+  text-align: center;
+  font-style: italic;
+  font-size: small;
+  padding-bottom: 16px;
+}
+
+.caption-img>p {
+  padding-top: 0;
+  margin-bottom: 4px;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: 'CMU-Serif', system-ui, -apple-system, BlinkMacSystemFont,
+  'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans',
+  'Droid Sans', 'Helvetica Neue', 'Noto Sans', sans-serif;
+}
```

Update fonts to serif and change color
```suggestion
h1, h2, h3, h4, h5, h6 {
  font-family: 'CMU-Serif', 'Times New Roman', Garamond, Georgia, serif;
  color: var(--ifm-color-gray-1000);
}
```